app/controllers/api/v1/ratings_controller.rb

```diff
@@ -0,0 +1,40 @@
+module Api
+  module V1
+    class RatingsController < ApplicationController
+      before_action :set_recipe, only: %i[create update]
+      before_action :set_rating, only: %i[update]
+
+      def create
+        @rating = @recipe.rating.new(rating_params)
+        if @rating.save
+          render json: @rating
+        else
+          render json: { errors: @rating.errors.to_s }, status: :unprocessable_entity
+        end
+      end
+
+      def update
+        if @rating.update(rating_params)
+          render json: @rating
+        else
+          render json: { errors: @rating.errors.to_s }, status: :unprocessable_entity
+        end
+      end
+
+      private
+
+      def rating_params
+        params.require(:rating).permit(%i[rating.recipe_id value average_rating]) # user_id
```

Comment:
Why are you allowing anything except `value`?
```suggestion
        params.require(:rating).permit(%i[value]) # user_id
```

Reply:
Please remove comments and dead code